wimm.Secundatives.UnitTests/Extensions/ResultMapping_Test.cs

```diff
@@ -0,0 +1,186 @@
+﻿using System.Threading.Tasks;
+using Xunit;
+
+namespace wimm.Secundatives.UnitTests.Extensions
+{
+    public class ResultMapping_Test
+    {
+        [Fact]
+        public void MapResult_IsSuccessAndFuncReturnsValue_ReturnsResultOfFunc()
+        {
+            var underTest = ConstructWith("doot");
+            var res = underTest.Map(x => "deet");
+
+            Assert.Equal("deet", res.Value);
+        }
+
+        [Fact]
+        public void MapResult_IsErrorAndFuncReturnsValue_ReturnsError()
+        {
+            var underTest = new Result<string, TestError>(TestError.Sadness);
+
+            var res = underTest.Map(x => "deet");
+
+            Assert.True(res.IsError);
```

Comment:
The is error test is redundant since accessing Error on a non-error will throw

Reply:
Was more about being clear in the assertions than anything. Want to fail the assert not just throw